sql/parser/sql.y

```diff
@@ -861,14 +861,14 @@ iso_level:
     // Mapped to the closest supported isolation level.
     $$ = SnapshotIsolation
   }
-| REPEATABLE READ
+| SNAPSHOT
   {
-    // Mapped to the closest supported isolation level.
     $$ = SnapshotIsolation
   }
-| SNAPSHOT
+| REPEATABLE READ
   {
-    $$ = SnapshotIsolation
+    // Mapped to the closest supported isolation level.
```

Comment:
Why make it an error? Our `SERIALIZABLE` provides a superset of the guarantees that the standard specifies for `REPEATABLE READ`, so we might as well support all four standard isolation levels with their closest equivalent. Other databases do the same, e.g. `READ UNCOMMITTED` is often implemented as a synonym for `READ COMMITTED` and no one minds that they never actually see dirty data.